R/missingLevelsTrain.R

```diff
@@ -0,0 +1,15 @@
+#' @title Set missing factor levels in test data to NA
+#' @description
+#' Sets missing factor levels in test data to NA to avoid prediction error for lm models
+#' @param model A lm* model fitted on training data.
+#' @param test.data [\code{data.frame}] \cr
+#'   Data set for prediction.
+#' @return A \code{data.frame} with matching factor levels of test data and fitted model
+missingLevelsTrain = function(model, test.data) {
+  lmod = getLearnerModel(model)
+  UseMethod("missingLevelsTrain", lmod)
+}
+
+missingLevelsTrain.default = function(model, test.data) {
+  return(test.data)
```

Comment:
Hi @larskotthoff, could you be a bit more precise here? What could go possibly wrong here? I see no case since this S3 method only applies for models of the lm* family and `.default` will apply for all others. 

Do you mean I should issue a message that it does nothing for other learners? I thinks that is not needed since the aim here was to only implement it for lm* models.